### PR TITLE
added drag css class

### PR DESCRIPTION
**Pull request by Yosabba**
https://app.asana.com/0/1202587816478003/1202587816478014

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -29,11 +29,16 @@
 }
 
 .draggable-element,
-.div-element {
+.div-element {
```
Comment (rbuchmeier) [inline src/App.css]: Let's remove the unnecessary whitespace.

**Review by rbuchmeier — CHANGES_REQUESTED** [review]
Looks great! One comment.

File: src/App.css (hunk 1 above)
Comment (Yosabba) [reply src/App.css]: https://github.com/Yosabba/page-builder/pull/1/commits/db7063dd580d61d6c473695c543f73e2ef17d8f4

**Review by rbuchmeier — APPROVED** [review]
